webapp/certification/helpers.py

```diff
@@ -0,0 +1,29 @@
+def get_pagination_page_array(page, total_pages):
```

Comment:
I don't think you need this, look at how we do pagination for CVE's and Tutorials, there are shared templates to achieve this.

Reply:
But the difference is that in this one we are doing it with `pages` instead of `limit` and `offset`?

Reply:
There are two templates on the codebase one of them uses pages the other uses offsets. I like offset + limit. I actually don't know if we prefer one or the other maybe @nottrobin knows?

Reply:
I am sure robin prefers the offset + limt. But I thought that one is harder for Front-end needs more calculation, but I guess @bethcollins92 take the code from [CVE](https://github.com/canonical-web-and-design/ubuntu.com/blob/master/templates/security/cve/index.html#L88).